src/components/Nav.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import './Nav.css';
```

Comment:
Please ensure that the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/165498151-206ef899-4c27-4941-bc93-5cc0be33a351.png) | ![image](https://user-images.githubusercontent.com/10332499/165498059-71dc35c0-3ce0-4435-b42d-a304a8fd95aa.png)